### ISSUE TITLE
Have you seen maidenone/RobotCreator

### ISSUE THREAD

**Issue by luzpaz** [Dave-Elec/freecad_to_gazebo#1]
https://github.com/maidenone/RobotCreator

**Comment by Dave-Elec** [comment]
Yes, I have seen that. The last commit is in 2017. And it has some issues.

**Comment by luzpaz** [comment]
CC @maidenone

**Comment by maidenone** [comment]
fun to see that someone else is working on this.
i am currently "porting" the solidworks files in this project to freecad: https://github.com/open-dynamic-robot-initiative/open_robot_actuator_hardware

my plan is to then continue with my freecad to gazebo/ignition workbench. but i lack spare time so it might take some time.

**Comment by movefasta** [comment]
> my plan is to then continue with my freecad to gazebo/ignition workbench. but i lack spare time so it might take some time.

Lookout at fork - https://github.com/RoboStack/RobotCreator
Maybe some changes can be merged.
